transcode/transcode.go

```diff
@@ -231,7 +231,7 @@ func transcodeSegment(segment segmentInfo, streamName, manifestID string, transc
 	}
 
 	for _, stats := range transcodedStats {
-		stats.BitsPerSecond = uint32(float64(stats.Bytes) * 8000.0 / float64(stats.DurationMs))
+		stats.BitsPerSecond = uint32(float64(stats.Bytes) * 8000.0 / float64(stats.DurationMs/1000))
```

Comment:
Had to update our bitrate calculation, which is bits / second but uses a millisecond variable to calculate it

Reply:
```suggestion
		stats.BitsPerSecond = uint32(float64(stats.Bytes) * 8000.0 / float64(stats.DurationMs/time.Second))
```

WDYT?